app/models/job.rb

```diff
@@ -1,5 +1,5 @@
 class Job < ApplicationRecord
-  include_concern 'StateMachine'
+  include StateMachine
```

Comment:
This should stay include_concern for all the reasons defined in [include_concern](https://github.com/ManageIQ/manageiq/blob/15b4acf10d915780f1651c170a31c11487efe8a7/lib/extensions/as_include_concern.rb#L4-L47)